src/main/java/org/gridsuite/loadflow/server/LoadFlowController.java

```diff
@@ -77,4 +71,13 @@ public ResponseEntity<List<String>> getProviders() {
     public ResponseEntity<String> getDefaultLoadflowProvider() {
         return ResponseEntity.ok().body(loadFlowService.getDefaultProvider());
     }
+
+    @GetMapping(value = "/specific-parameters")
+    @Operation(summary = "Get all existing loadflow specific parameters for a given provider, or for all of them")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The loadflow model-specific parameters")})
+    public ResponseEntity<Map<String, List< com.powsybl.commons.parameters.Parameter>>> getSpecificLoadflowParameters(
```

Comment:
why is it fully qualified?

Reply:
I have to use both io.swagger.v3.oas.annotations.Parameter and com.powsybl.commons.parameters.Parameter.
I dont know how to handle this differently.